Parser/infinitySDLoggerParse.m

```diff
@@ -1,181 +1,184 @@
-function sample_data = infinitySDLoggerParse( filename, mode )
-%infinitySDLogger Parses a .csv data file retrieved from a JFE Infinity ACLW-USB logger.
-%
-%
-% Inputs:
-%   filename    - Cell array containing the name of the file to parse.
-%   mode        - Toolbox data type mode.
-%
-% Outputs:
-%   sample_data - Struct containing imported sample data.
-%
-% Author : Guillaume Galibert <guillaume.galibert@utas.edu.au>
-
-%
-% Copyright (C) 2017, Australian Ocean Data Network (AODN) and Integrated 
-% Marine Observing System (IMOS).
-%
-% This program is free software: you can redistribute it and/or modify
-% it under the terms of the GNU General Public License as published by
-% the Free Software Foundation version 3 of the License.
-%
-% This program is distributed in the hope that it will be useful,
-% but WITHOUT ANY WARRANTY; without even the implied warranty of
-% MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-% GNU General Public License for more details.
-
-% You should have received a copy of the GNU General Public License
-% along with this program.
-% If not, see <https://www.gnu.org/licenses/gpl-3.0.en.html>.
-%
-  
-% ensure that there is one or two arguments
-narginchk(1,2);
-
-% and that it is a cell array of strings
-if ~iscellstr(filename), error('filename must be a cell array of strings'); end
-
-% only one file supported currently
-filename = filename{1};
-if ~ischar(filename), error('filename must contain a string'); end
-
-% open the file, and read in the data
-try
-    fid = fopen(filename, 'rt');
-    rawText = textscan(fid, '%s', 'Delimiter', '');
-    fclose(fid);
-    
-    [header, iData] = readHeader(rawText{1});
-    data            = readData(rawText{1}(iData:end));
-catch e
-    if fid ~= -1, fclose(fid); end
-    rethrow(e);
-end
+function sample_data = infinitySDLoggerParse(filename, mode)
+    %infinitySDLogger Parses a .csv data file retrieved from a JFE Infinity ACLW-USB logger.
```

Comment:
You've introduced a new indent on the whole function which makes it difficult to read the diff. Was that necessary?